apps/tlon-mobile/src/screens/ChatListScreen.tsx

```diff
@@ -81,7 +81,7 @@ export default function ChatListScreen(
     <ContactsProvider contacts={contacts ?? []}>
       <View backgroundColor="$background" flex={1}>
         <ScreenHeader
-          title={<Icon type="TBlock" size="$xl" flex={1} rotate="-6deg" />}
+          title={<Icon type="TBlock" size="$m" flex={1} rotate="-6deg" />}
```

Comment:
Given the original image I was just about to comment that the block was a little large, this feels like it'll do the trick as long as it's close to designs / proportional to the + icon

Reply:
Yes, amended. Thank you